openapi/src/main/java/io/helidon/openapi/OpenAPISupport.java

```diff
@@ -182,7 +305,7 @@ private void prepareResponse(ServerRequest req, ServerResponse resp) {
             resp.status(Http.Status.OK_200);
             resp.headers().add(Http.Header.CONTENT_TYPE, resultMediaType.toString());
             resp.send(openAPIDocument);
-        } catch (IOException ex) {
+        } catch (Exception ex) {
```

Comment:
The old version would let `RuntimeException`s be thrown here; not sure if you intended to catch them as well.

Reply:
The intent of the change is to catch _any_ `Exception` (and convert it into a 500 response with a brief explanation sent back to the client).